flocker/control/_persistence.py

```diff
@@ -193,6 +193,36 @@ def upgrade_from_v4(cls, config):
             }
         return dumps(decoded_config)
 
+    @classmethod
+    def upgrade_from_v5(cls, config):
+        """
+        Migrate a v5 JSON configuration to v6.
+
+        :param bytes config: The v5 JSON data.
+        :return bytes: The v6 JSON data.
+        """
+        decoded_config = loads(config)
+        decoded_config[u"version"] = 6
+        try:
+            nodes = decoded_config[u"deployment"][u"nodes"]
+        except KeyError:
+            pass
+        else:
+            new_node_values = []
+            for node in nodes[u"values"]:
+                uuid = node[0]
+                applications = node[1][u"applications"][u"values"]
+                for app in applications:
+                    app[1].update({u'swappiness': None})
+                new_node = node[1]
+                new_node[u"applications"][u"values"] = applications
+                new_node_values.append((uuid, new_node))
+            decoded_config[u"deployment"][u"nodes"] = {
+                u"values": new_node_values,
+                _CLASS_MARKER: "PMap"
+            }
+        return dumps(decoded_config)
+
```

Comment:
:question: You probably don't need this upgrade script, since `swappiness` is an optional attribute in the model (`mandatory=False`)